httpserver/nbproject/project.xml

```diff
@@ -153,7 +153,7 @@
             <public-packages/>
             <class-path-extension>
                 <runtime-relative-path>ext/webserver.jar</runtime-relative-path>
-                <binary-origin>external/tomcat-webserver-3.2.jar</binary-origin>
+                <binary-origin>release/modules/ext/webserver-3.2.4.jar</binary-origin>
```

Comment:
This should be `external/webserver-3.24.jar` (the relative path to the downloaded binary)

Reply:
Solved